lib/sqlalchemy/sql/elements.py

```diff
@@ -2217,13 +2259,13 @@ def _select_iterable(self) -> _SelectIterable:
     _allow_label_resolve = False
 
     @property
-    def _is_star(self):
+    def _is_star(self) -> bool:
```

Comment:
Sometimes when there is an attribute overridden with a `@property` mypy would complain about it, despite it seemingly being a good practice. This might be due to a [mypy bug](https://github.com/python/mypy/issues/4125), however [here](https://stackoverflow.com/questions/58349417/how-to-annotate-attribute-that-can-be-implemented-as-property) a guy makes a point about how this might become a legitimate bug and suggests a few fixes. All of them require modification of the source, and I cannot find any other fixes - have to ask the author

Reply:
Found a [thread](https://github.com/python/mypy/issues/12440) from Mike with a bit more info on mypy's relationship with `@property`. It also looks related to [this comment](https://github.com/GodelTech/sqlalchemy/pull/5#discussion_r1065929005) regarding incompatible definitions in base classes.